<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_background">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_textPoz">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>16</width>
      <height>512</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
    <property name="value">
     <number>256</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_textPoz">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>512</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
    <property name="value">
     <number>256</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadImage">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Incarca imaginea de fundal</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_textSize">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>249</y>
      <width>20</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_textBold">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>249</y>
      <width>20</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_textWrite">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Scrie textul</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_coordX">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_coordY">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>510</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_size">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bold">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>420</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bold:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_textColor">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Alege culoarea textului</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_color">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>290</y>
      <width>51</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>445</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvare</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuT03">
    <property name="title">
     <string>T03</string>
    </property>
   </widget>
   <addaction name="menuT03"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
